app/views/layouts/_footer.html.haml

```diff
@@ -4,7 +4,9 @@
       .footer-copyright
         &copy; 2011&ndash;
         = Time.new.year
-        = _("openSUSE contributors")
+        = _('openSUSE contributors')
       .list-inline
-        = link_to _("Source Code"), "https://github.com/openSUSE/news-o-o", class: "list-inline-item"
-        = link_to _("License"), "https://github.com/openSUSE/news-o-o/blob/master/LICENSE", class: "list-inline-item"
+        - github_link = 'https://github.com/openSUSE/news-o-o'
```

Comment:
news-o-o?

Reply:
Oh, that's something I missed when copying from elsewhere, gonna correct that